README.md

```diff
@@ -243,17 +243,32 @@ node app.js [options]
+|                    | LISK_CONSOLE_LOG_LEVEL | consoleLogLevel          | Log level for console output                                                                                                                                                      |
+|                    | LISK_CACHE_ENABLED     | cacheEnabled             | Enable or disable cache. Must be set to true/false                                                                                                                                |
+| --database<br> -d  | LISK_DB_NAME           | db.database              | PostgreSQL database name to connect                                                                                                                                               |
+|                    | LISK_DB_HOST           | db.host                  | PostgreSQL database host name                                                                                                                                                     |
+|                    | LISK_DB_PORT           | db.port                  | PostgreSQL database port                                                                                                                                                          |
+|                    | LISK_DB_USER           | db.user                  | PostgreSQL database username to connect                                                                                                                                           |
+|                    | LISK_DB_PASSWORD       | db.password              | PostgreSQL database password to connect                                                                                                                                           |
+| --peers<br> -p     | LISK_PEERS             | peers.list               | Comma separated list of peers to connect in the format `192.168.99.100:5000,172.169.99.77:5000`                                                                                   |
+|                    | LISK_API_PUBLIC        | api.access.public        | Enable or disable public access of http API. Must be set to true/false                                                                                                            |
+|                    | LISK_API_WHITELIST     | api.access.whiteList     | Comma separated list of IPs to enable API access. Format `192.168.99.100,172.169.99.77`                                                                                           |
+|                    | LISK_FORGING_DELEGATES | forging.delegates        | Comma separated list of delegates to load in the format `publicKey|encryptedPassphrase,publicKey2|encryptedPassphrase2`                                                           |
```

Comment:
I think you have to escape that, it does show properly in the rendered README file.